dusty/commands/test.py

```diff
@@ -143,10 +143,8 @@ def _run_tests_with_image(client, expanded_specs, app_or_lib_name, test_command,
 
     for service_container in previous_container_names:
         log_to_client('Stopping service container {}'.format(service_container))
-        try:
-            client.stop(service_container, timeout=1)
-        except Exception as e:
-            log_to_client('Exception stopping service container {}: {}'.format(service_container, str(e)))
+        client.kill(service_container)
+        client.remove_container(container=service_container)
```

Comment:
Struggling to remember why we didn't do this in the first place. Maybe it was because we were thinking we'd share service containers across suites then?